index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Final projects</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <header>
+  <nav class="nav__bar" id="navbar">
+      <ul>
+        <li><a class="nav__links" href="#welcome-section">About</a></li>
+        <li><a class="nav__links" href="#projects">work</a></li>
+        <li><a class="nav__links" href="#cantact">cantact</a></li>
+      </ul>
+  </nav>
+  <section class="section__first" id="welcome-section">
+      <h1 class="section__h1">Hey I am Roya Jamal </h1>
+      <p class="section__p"> A web developer</p>
+  </section>
+  </header>
```

Comment:
- The header is parent and the child elements should be 2 spaces ahead of the parent so add 2 spaces to the header children.
- The ul and section child has 4 spaces change them to 2 spaces.